.gitignore

```diff
@@ -4,6 +4,4 @@ dbt_modules/
 logs/
 env/
 dbt_packages/
-.DS_Store
-integration_tests/package-lock.yml
-integration_tests/.DS_Store
```

Comment:
why remove these?

Reply:
These lines in the gitignore are actually redundant. Since we already have `package-lock.yml` and `.DS_Store` listed in the root directory gitignore, that means they will be ignored in all subdirectories as well.